Source/WTF/wtf/haiku/RunLoopHaiku.cpp

```diff
@@ -27,11 +27,25 @@
 #include "wtf/RunLoop.h"
 
 #include <Application.h>
+#include <errno.h>
```

Comment:
includes from the Be API and from the C standard shouldn't be mixed together in the same block, I think?